setup.py

```diff
@@ -47,15 +47,16 @@ def run_tests(self):
 cmdclass.update({"test": PyTest})
 
 install_requirements = [
-    "pandas",
+    "pandas>=0.24.0",
     "pandas-datapackage-reader",
     "f90nml",
     "pyam-iamc @ git+https://github.com/IAMconsortium/pyam.git@master",
     "PyYAML",
     "expectexception @ git+https://github.com/thedataincubator/expectexception.git@684560f4dfb8cb69eb0693a1605ec0e81fd9b42a",
      "ipywidgets",
      "appmode",
-     "openscm @ git+https://github.com/openclimatedata/openscm.git@add-openscm-dataframe"
+     "openscm @ git+https://github.com/openclimatedata/openscm.git@add-openscm-dataframe",
+     "seaborn"
```

Comment:
So any dependencies used in the notebooks get added as explicit dependencies? I mean this already gets installed via seaborn, but I would put the extra dependencies from notebooks as a separate extra requirement target.

Reply:
@rgieseke and I talked about this a bit, see https://github.com/openclimatedata/pymagicc/pull/198#issuecomment-457640697

I still prefer to have these as explicit dependencies, but happy to change